apps/condo/domains/user/access/User.js

```diff
@@ -4,22 +4,32 @@
 const access = require('@core/keystone/access')
 
 async function canReadUsers ({ authentication: { item: user } }) {
+
     if (!user || !user.id) return false
-    if (user.isAdmin) return true
-    return true
+
+    if (user.isAdmin) {
+        return {}
```

Comment:
@pahaz should we return `true` for `user.isAdmin` permissions both in `canReadUsers` & `canManageUsers` guards?